src/Dropzone/templates/form_theme.html.twig

```diff
@@ -1,24 +1,17 @@
 {% block dropzone_widget -%}
-    {%- set dataController = (attr['data-controller']|default('') ~ ' symfony--ux-dropzone--dropzone')|trim -%}
-    {%- set attr = attr|merge({'data-controller': '', class: (attr.class|default('') ~ ' dropzone-input')|trim}) -%}
+{%- set dataController = (attr['data-controller']|default('') ~ ' symfony--ux-dropzone--dropzone')|trim -%}
```

Comment:
Could you revert the indentation as it was ? 😄  

It's to avoid unnecessary changes, and ease the review / maintain process (now and after i mean)